src/common/meta/src/error.rs

```diff
@@ -766,7 +780,9 @@ impl ErrorExt for Error {
             | SerializeToJson { .. }
             | DeserializeFromJson { .. } => StatusCode::Internal,
 
-            ValueNotExist { .. } => StatusCode::Unexpected,
+            NoLeader { .. } => StatusCode::TableUnavailable,
+
+            ValueNotExist { .. } | ConsistencyGuardConflict { .. } => StatusCode::Unexpected,
```

Comment:
I think `ConsistencyGuardConflict` should return `TableUnavailable` too.

Reply:
The upper layer will retry if the status is `TableUnavailable`, which is not the expected behavior. When a `ConsistencyGuardConflict` occurs, users need to manually repair the table using a repair procedure (we will provide later)